### ISSUE TITLE
New features, e.g. similar to Angular/React

### ISSUE THREAD

**Issue by seagullmouse** [denofevil/AureliaStorm#35]
Firstly, thanks so much for this plugin: it's been invaluable for me!

Do you intend to add more functionality/features to this plugin?

For inspiration I've just been looking at the [WebStorm 2019.1 new features](https://www.jetbrains.com/webstorm/whatsnew/). In general it looks like IntelliJ is becoming more framework 'aware', i.e. views and models are aware of each other, views are aware of components etc. It would be wonderful if we could achieve something like this for Aurelia

A few things specifically spring to mind from my usage.

1. Making views aware of locations of components, i.e. X would autocomplete in "<require from="X" >
2. Auto-add the require statement when using a component, e.g. typing `<my-component>` would suggest adding the require for that component.
3. Refactoring of component locations should update locations in require tags.
4. Improve inspections to support valid view syntax
e.g. `<td>${faker.random.number({min:0, max:10000})}</td>` currently throws a `Syntax error: ',expected'`

Thanks again for your work on this!

**Comment by denofevil** [comment]
4 is fixed via 95d0e650c63b2a19f895b6286c16202e65933404
3 is #14, #3
I think that 1 would be covered by #15

**Comment by seagullmouse** [comment]
Another useful addition would be syntax highlighting for repeaters.
e.g.
```
<template repeat.for="groupType of groupTypes">
${groupType}
</template>
```
- The reference to groupTypes in the view model is not recognised.
- The reference to the new variable `groupType` in the view is not recognised.

Thanks again
